plugins/rikki/heroeslounge/routes.php

```diff
@@ -31,8 +31,9 @@
     Route::get('matches/withApprovedCastBetween/{startdate}/{enddate}','Rikki\Heroeslounge\Http\Match@withApprovedCastBetween');
     Route::get('matches/{id}/games','Rikki\Heroeslounge\Http\Match@games');
     Route::get('matches/{id}/replays','Rikki\Heroeslounge\Http\Match@replays');
-    Route::get('matches/{date}/{tz1}/{tz2?}','Rikki\Heroeslounge\Http\Match@getTodaysMatches')->where('date', 'today|\d{4}-\d{2}-\d{2}');
-    Route::get('matches/{date}','Rikki\Heroeslounge\Http\Match@getTodaysMatches')->where('date', 'today|\d{4}-\d{2}-\d{2}');
+    Route::get('matches/today/{tz1?}/{tz2?}','Rikki\Heroeslounge\Http\Match@getTodaysMatches');
+    Route::get('matches/byDate/{date}/{tz1?}/{tz2?}','Rikki\Heroeslounge\Http\Match@getMatchesByDate');
```

Comment:
For consistency with `getTodaysMatches`, the other should be called `getByDateMatches`.

Alternatively, change to `getMatchesForToday` and leave other as `getMatchesByDate`